src/foam/dao/index/ValueIndex.java

```diff
@@ -0,0 +1,63 @@
+/**
+ * @license
+ * Copyright 2017 The FOAM Authors. All Rights Reserved.
+ *     http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
I will fix it, but it's the same format as some of the files you merged here(All the `Plan`s Ones) so I just copied and pasted the headers from one of this Plans merged before: https://github.com/foam-framework/foam2/pull/518/files